subsys/lw_sched/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if (CONFIG_LW_SCHEDULER)
```

Comment:
This isn't needed, as this directory is added with `add_subdirectory_ifdef(CONFIG_LW_SCHEDULER lw_sched)`

Reply:
Removing ...